runtime/stdlib/contracts/test.cdc

```diff
@@ -346,4 +447,27 @@ pub contract Test {
         })
     }
 
+    /// Asserts that the result of an executed operation, such as
+    /// scripts & transactions, has errored and the error message
+    /// contains the given sub-string.
+    ///
+    pub fun assertError(_ result: {Result}, substr: String) {
```

Comment:
maybe rename the variable `substr` to `errorMessage`

Reply:
Good idea :ok_hand: 
Fixed in https://github.com/onflow/cadence/pull/2696/commits/600ed65a245e0d67a65f677c60c707fa047dfff1